src/components/Btn/Btn.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from "@mui/material";
+
+import "./Btn.css";
```

Comment:
for styles if it is only one file in folder then use naming like `style.css` or `styles.css`